website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,8 +20,27 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## June 2024
 
-- **New:** [Job warnings](/docs/deploy/job-notifications) are now GA. Previously, you could receive email or Slack alerts about your jobs when they succeeded, failed, or were canceled. Now with the new **Warns** option, you can also receive alerts when jobs have encountered warnings from tests or source freshness checks during their run. This gives you more flexibility on _when_ to be notified. 
+#### dbt Semantic Layer
+- **New:** Introduced new granularity support for cumulative metrics in MetricFlow. Granularity options for cumulative metrics are slightly different than granularity for other metric types. For other metrics, we use the date_trunc function to implement granularity. However, because cumulative metrics are non-additive (values can't be added up), we can't use the date_trunc function to change their time grain granularity. 
+  
+  Instead, we use the `first()`, `last()`, and `avg()` aggregation functions to aggregate cumulative metrics over the requested period. By default, we take the first value of the period. You can change this behavior by using the `period_agg` parameter. For more information, refer to [Granularity options for cumulative metrics](/docs/build/cumulative#granularity-options).
+- **New:** Added support for <Term id="predicate-pushdown"/> SQL optimization in MetricFlow. We will now push down categorical dimensions filters to the metric source table. Previously filters we're applied after we selected from the metric source table. This change helps reduce full table scans on certain query engines.
+- **New:** Enabled cache to pickup where filters that are included in saved queries.
+- **Enhancement:** Updated MetricFlow to use the new `cumulative_type_params` fields for cumulative metrics.
```

Comment:
removing per courtney's sugg here: https://linear.app/dbt-labs/issue/SL-2364/add-cumulative-type-params-to-yaml-spec
```suggestion
```